.github/workflows/prettify-code.yml

```diff
@@ -11,8 +11,8 @@ jobs:
       steps:
          - name: Checkout Repository
            uses: actions/checkout@v4
+         - name: install deps
+           run: npm install
```

Comment:
changed to running our script since the action was using a different version of prettier, causing this check to fail